cmd/terramate/cli/backend_cfg_gen_test.go

```diff
@@ -126,7 +116,9 @@ func TestBackendConfigGeneration(t *testing.T) {
 					config: `terramate {
   required_version = "~> 0.0.0"
   backend "sometype" {}
-}`,
+}
+
+stack {}`,
```

Comment:
I only put `stack {}` on the actual stack changes. Now directories that are not stacks have no stack block.

Reply:
@katcipis I have an idea to improve this with the sandbox mini-lang but would like to talk to you first.

Reply:
that would be great, but lets discuss/improve on a follow up PR :heart: